### PR TITLE
Share commercial page targeting for server-to-server integrations

### PR DESCRIPTION
**Pull request by rich-nguyen**
Server-to-server integrations like sonobi-appnexus need page targeting made available before Googletag has even loaded.

### CODE REVIEW

**Comment by rich-nguyen** [comment]
[Sonobi-to-AppNexus integration needs access to the page targeting.](https://trello.com/c/aAGS1rRL/231-sonobi-to-appnexus-integration-needs-access-to-the-page-targeting)

File: static/src/javascripts-legacy/projects/commercial/modules/build-page-targeting.js [hunk 1]
```diff
@@ -163,8 +165,8 @@ define([
         }
     }
 
-    return function (opts) {
-        var win         = (opts || {}).window || window;
+    return once(function () {
```
Comment (rich-nguyen) [inline static/src/javascripts-legacy/projects/commercial/modules/build-page-targeting.js]: `once` to the rescue again!

**Review by regiskuckaertz — APPROVED** [review]

**Comment by regiskuckaertz** [comment]
Memoization and unit tests don't go along very often 😭

File: static/test/javascripts-legacy/spec/commercial/build-page-targeting.spec.js [hunk 2]
```diff
@@ -18,6 +18,11 @@ define([
             injector = new Injector();
 
         beforeEach(function (done) {
+
+            injector.mock('lodash/functions/once', function once(func) {
+                return func;
```
Comment (regiskuckaertz) [inline static/test/javascripts-legacy/spec/commercial/build-page-targeting.spec.js]: 😆

**Pull request merged**